@xen-orchestra/web-core/lib/components/table/ColumnHeader.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <MenuList :disabled="disabled" placement="bottom-start" shadow>
```

Comment:
You can use the shorthand syntax here:
```suggestion
  <MenuList :disabled placement="bottom-start" shadow>
```